clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -278,12 +183,8 @@ public EhcacheEntityResponse invoke(ClientDescriptor clientDescriptor, EhcacheEn
       if (message instanceof EhcacheOperationMessage) {
         EhcacheOperationMessage operationMessage = (EhcacheOperationMessage) message;
         EhcacheMessageType messageType = operationMessage.getMessageType();
-        if (isStoreOperationMessage(messageType)) {
-          return invokeServerStoreOperation(clientDescriptor, (ServerStoreOpMessage) message);
-        } else if (isLifecycleMessage(messageType)) {
+        if (isLifecycleMessage(messageType)) {
           return invokeLifeCycleOperation(clientDescriptor, (LifecycleMessage) message);
-        } else if (isStateRepoOperationMessage(messageType)) {
-          return invokeStateRepositoryOperation(clientDescriptor, (StateRepositoryOpMessage) message);
         } else if (isPassiveReplicationMessage(messageType)) {
```

Comment:
I guess `EhcacheActiveEntity` should never be getting any `PassiveReplicationMessage` since execution strategy prohibits that, right ?